pandas/core/frame.py

```diff
@@ -3030,6 +3059,364 @@ def fillna(self, value=None, method=None, axis=None, inplace=False,
+                  'y' with 'z'. To use a dict in this way the ``value``
+                  parameter should be ``None``.
+                - Alternatively, a dict can specify that different values
+                  should be replaced in different columns. For example,
+                  {'a': 1, 'b': 'z'} looks for the value 1 in column 'a' and
+                  the value 'z' in column 'b' and replaces these values with
+                  whatever is specified in ``value``. The ``value`` parameter
+                  should not be ``None`` in this case. You can treat this as a
+                  special case of passing two lists except that you are
+                  specifying the column to search in.
+                - Nested dictionaries, e.g., {'a': {'b': np.nan}}, are read as
+                  follows: look in column 'a' for the value 'b' and replace it
+                  with NaN. The ``value`` parameter should be ``None`` to use
+                  a nested dict in this way. You can nest regular expressions
+                  as well. Note that column names (the top-level dictionary
+                  keys in a nested dictionary) **cannot** be regular
+                  expressions.
+
+            * None:
+
+                - This means that the ``regex`` argument must be a string,
+                  compiled regular expression, or list, dict, ndarray or Series
+                  of such elements. If ``value`` is also ``None`` then this
+                  **must** be a nested dictionary or ``Series``.
+
+            See the examples section for examples of each of these.
+        value : scalar, dict, list, str, regex, default None
+            Value to replace any values matching ``to_replace`` with.
+            Alternatively, a dict of values specifying which value to use for
+            each column (columns not in the dict will not be filled). Regular
+            expressions, strings and lists or dicts of such objects are also
+            allowed.
+        inplace : boolean, default False
+            If True, in place. Note: this will modify any
+            other views on this object (e.g. a column from a DataFrame).
+            Returns the caller if this is True.
+        limit : int, default None
+            Maximum size gap to forward or backward fill
+        regex : bool or same types as `to_replace`, default False
+            Whether to interpret ``to_replace`` and/or ``value`` as regular
+            expressions. If this is ``True`` then ``to_replace`` *must* be a
+            string. Alternatively, this could be a regular expression or a list,
+            dict, or array of regular expressions in which case ``to_replace``
+            must be ``None``.
+        method : string, optional, {'pad', 'ffill', 'bfill'}
+            The method to use when for replacement, when ``to_replace`` is a
+            ``list``.
+
+        See Also
+        --------
+        :func:`DataFrame.fillna` : Fill NA/NaN values
```

Comment:
You do't need the

   :func:`

. Just DataFrame.fillna should be enough for numpydoc.